system/include/emscripten/val.h

```diff
@@ -176,6 +173,11 @@ T fromGenericWireType(double g) {
   return BindingType<T>::fromWireType(wt);
 }
 
+template<>
+inline void fromGenericWireType<void>(double g) {
```

Comment:
I don't quite follow this. Why is this only needed for doubles?

Reply:
Github diff hides it, but see `fromGenericWireType` above. This simply specialises that existing definition.

They take `double` because that's what generic wire type is. Arguably this should take `EM_GENERIC_WIRE_TYPE` for clarity but I'm really trying to keep PRs small and isolated as per previous requests 😅

Reply:
OTOH... this is probably a reasonable exception. I'll just update both for clarity.